### PR TITLE
implemented modify and delete password features

### PR DESCRIPTION
**Pull request by DhruvMathur03**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -8,21 +8,16 @@
...
     print("Login")
     email = cli_ui.ask_string("Email")
     password = cli_ui.ask_password("Password")
-
     does_user_exist = database.find(
         'Users', f'email = "{email}" AND password="{password}"')
-
     if len(does_user_exist) > 0:
-        user_info = user.User(does_user_exist[0][0], email, database)
+        login.user_info = user.User(does_user_exist[0][0], email, database)
```
Comment (YashMathur) [inline main.py:20]: what's `login.user_info`? where is `login` defined?

Comment (DhruvMathur03) [reply main.py:20]: I found this way to use a variable defined in one function elsewhere

File: main.py [hunk 2]
```diff
@@ -7,17 +7,12 @@
 import clipboard
 
 database = db.DB("pugsey.db")
-logged_in_user = None
-user_info = None
```
Comment (YashMathur) [inline main.py:11]: You're using `user_info` now so add this line back. It should be declared as a global variable

**Review by YashMathur — APPROVED** [review]
Just fix that one comment and then you can merge it as long as all the features are working fine

**Pull request merged**